src/node_http_common.h

```diff
@@ -123,6 +127,55 @@ enum http_known_headers {
   HTTP_KNOWN_HEADER_MAX
 };
 
+static const std::unordered_set<size_t> http2_single_value_headers{
```

Comment:
you can do `static constexpr` like this: https://github.com/ada-url/ada/blob/main/src/helpers.cpp#L746

Reply:
```
Variable of non-literal type 'std::unordered_set<unsigned long>' cannot be defined in a constexpr function before C++2b

clang(constexpr_local_var_non_literal_type)
```

getting this from clang for this, any way around this? assume we arent doing the most modern standard yet?

Reply:
std::unordered_set by default can have any size. If you use `std::array` it will be a fixed size.

Reply:
whats best in terms of performance for search? ie `<container>.contains(<value>)`

Reply:
also, how can we calculate std::hash as constexpr (assuming we do need it here in the way that I've used it)

Reply:
yeah, std::hash is not constexpr, which is a bummer.